README.md

```diff
@@ -12,22 +12,115 @@
 ## Pre Deploy
 You need to store user management data in some table we use this table structure:
 ```
-CREATE TABLE IF NOT EXISTS users (
-  id bigserial primary key,
-  added_at timestamp with time zone DEFAULT now(),
-  email text not null UNIQUE,
-  password_hash text not null,
-  role text not null default 'user',
-  email_token uuid not null,
-  active boolean not null default false
-);
-
-CREATE TABLE IF NOT EXISTS refetch_tokens (
-  refetch_token uuid primary key,
-  user_id integer not null,
-  added_at timestamp with time zone DEFAULT now(),
-  FOREIGN KEY (user_id) REFERENCES users (id)
-);
+-- SCHEMA: user_management
```

Comment:
In the readme I think we should make it as simple as possible. I think we should not introduce a new schema. That is for advanced users and we can describe how to use a different schema than `public` when we write about the newly introduced `USER_MANAGEMENT_DATABASE_SCHEMA_NAME` environment variable.

Reply:
You are right. Done it in new commit. :wink: